release.sh

```diff
@@ -44,6 +45,9 @@ check() {
                 exit 1
             fi
         fi
+        if [ ! -z "$split" ] && [ "$split" != "SPLIT-RELEASE" ]; then
+            echo "Invalid entry in third column, must be SPLIT-RELEASE or non-existent."
```

Comment:
**Bikeshed alert**

If we're going for ALL_CAPS, shouldn't we use an underscore instead of a hyphen?